src/views/studio/studio-activity.jsx

```diff
@@ -8,6 +9,140 @@ import {activity} from './lib/redux-modules';
 import {activityFetcher} from './lib/fetchers';
 import Debug from './debug.jsx';
 
+import SocialMessage from '../../components/social-message/social-message.jsx';
+
+import './studio.scss';
+
+const getComponentForItem = item => {
+    switch (item.type) {
+    case 'addprojecttostudio':
+        return (
+            <SocialMessage
+                datetime={item.datetime_created}
+                iconSrc="/svgs/studio/activity-project.svg"
+                iconAlt="project activity icon"
+                imgClassName="studio-activity-icon"
+                key={item.id}
+            >
+                <FormattedMessage
+                    id="studio.activityAddProjectToStudio"
+                    values={{
+                        profileLink: (
+                            <a href={`/users/${item.actor_id}`}>
```

Comment:
A heads up, user links are /users/:username, not by id. I'd change this to `actor_username`, even if that isn't defined it, since it will be defined when we fix api. Same for the fixed string `username`, i'd change that to `{item.actor_username}`